pyk/src/pyk/kcfg.py

```
@@ -562,6 +618,9 @@ def remove_verified(self, source_id: str, target_id: str) -> None:
             raise ValueError(f'Edge is not verified: {(source_id, target_id)}')
         self._verified.remove((source_id, target_id))
 
+    def remove_alias(self, name: str) -> None:
+        self._aliases.pop(name)
```

```suggestion
    def remove_alias(self, alias: str) -> None:
        self._aliases.pop(alias)
```

---

This method should probably raise an exception if the alias is not present.